tools/theme/src/Theme.ts

```diff
@@ -304,9 +337,10 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
                 resolvedValue?: string,
                 actualValue?: string
             ): void => {
-                const themeModifier =
-                    this.theme && this.theme !== 'spectrum'
-                        ? `-${this.theme}`
+                const systemModifier =
+                    this.system &&
+                    !['spectrum', 'spectrum-two'].includes(this.system)
+                        ? `-${this.system}`
```

Comment:
I'm not sure this will allow for hot swapping between the `spectrum` and `spectrum-two`. Can you confirm with a test?

Reply:
_NO_ This will not